docs: fix spark workaround steps (#4365)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

    ```bash
    # e.g., this value can be spark-operator-1

>     # if your spark-operator appdeployment name doesn't contains spark, you need to adjust the grep command accordingly

Reviewer: ```suggestion
    # if your spark-operator AppDeployment name doesn't contain "spark", you must adjust the grep command accordingly
```

> 1. if you want to force a pod recreation, you can delete the old pod in `CrashLoopBackoff` by running

Reviewer 2: ```suggestion
1. If you want to force a pod recreation, you can delete the old pod in `CrashLoopBackoff` by running:
```

   ```bash
   # spark-operator is the default value
   # if you override the HelmRelease name in your override configmap, use that value here 
   export SPARK_OPERATOR_RELEASE_NAME=spark-operator

>    # We always want to only deploy 1 spark operator per cluster

Reviewer: ```suggestion
   # only one instance of spark operator should be deployed per cluster
```
